src/javascript/app/pages/trade/markets.jsx

```diff
@@ -299,8 +336,10 @@ class Markets extends React.Component {
         }
         const filter_markets = [];
         markets_all.map(([key, market]) => {
-            let found_for_market = false; // To check market contains any matching underlying.
+            let found_for_market = false;
             const filter_submarkets = {};
+            // To check market contains any matching underlying.
```

Comment:
Do we need to move this comment back to line 339?

Reply:
resolved